packages/account/src/Components/poi/idv-document-submit/idv-document-submit.jsx

```diff
@@ -79,6 +80,8 @@ const IdvDocumentSubmit = ({ handleBack, handleViewComplete, selected_country, i
     const validateFields = values => {
         const errors = {};
         const { document_type, document_number } = values;
+        const recurring_number_pattern = document_number.match(/([0-9])\1{3,}/g);
```

Comment:
this code is duplicated, by right you should be able to generalise it into a util function

Reply:
Moved it to utils and exported is as a function that requires document_number for input.